src/App.java

```diff
@@ -13,17 +13,76 @@ public void start() {
         printGreeting(userName);
         printInstructions();
         doMadlib();
-        System.out.println("hi");
+      
+
+        
     }
-    
+    public String getUserName(){
+  Scanner scanner = new Scanner(System.in);
+  System.out.print("Enter your name: ");
+   return scanner.nextLine();
+    }
+   
+
+
+
+
+   public void printGreeting(String userName){
+   System.out.println("Hello," + userName + "!");
+
+   }
+
+    public void printInstructions(){
+ System.out.println("To play this game you have to fill in the boxes with words that corresponds with what is asked.");
+
+    }
+          
+    public void doMadlib() {
```

Comment:
You need to document your source for externally learned information, such as ANSI color codes.

Reply:
You should also explain how they work, at least a bit, so I know that you understand the learning and didn't copy and paste somebody else's code with no real gain.